<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NYCMDLNMUHMAC.local" name="Default test" tests="1" failures="0" timestamp="11 Nov 2017 03:46:00 GMT" time="45.534" errors="0">
  <testcase name="@BeforeTest setup" time="0.0" classname="com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails">
    <failure type="org.openqa.selenium.WebDriverException" message="Timed out waiting 45 seconds for Firefox to start.
Build info: version: &amp;apos;3.6.0&amp;apos;, revision: &amp;apos;6fbf3ec767&amp;apos;, time: &amp;apos;2017-09-27T15:28:36.4Z&amp;apos;
System info: host: &amp;apos;NYCMDLNMUHMAC.local&amp;apos;, ip: &amp;apos;2604:2000:a806:6f00:f4cb:6af:487f:3aab&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.11.4&amp;apos;, java.version: &amp;apos;1.8.0_60&amp;apos;
Driver info: driver.version: FirefoxDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: Timed out waiting 45 seconds for Firefox to start.
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T15:28:36.4Z'
System info: host: 'NYCMDLNMUHMAC.local', ip: '2604:2000:a806:6f00:f4cb:6af:487f:3aab', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.8.0_60'
Driver info: driver.version: FirefoxDriver
	at org.openqa.selenium.firefox.XpiDriverService.waitUntilAvailable(XpiDriverService.java:112)
	at org.openqa.selenium.firefox.XpiDriverService.start(XpiDriverService.java:97)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:217)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:140)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:98)
	at com.framework.automation.framework.testBase.TestBase.selectBrowser(TestBase.java:41)
	at com.framework.automation.framework.testBase.TestBase.init(TestBase.java:23)
	at com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails.setup(VerifyloginWithInvalidCredentails.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.TestRunner.beforeRun(TestRunner.java:656)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://localhost:34324/hub/status] to be available after 45005 ms
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)
	at org.openqa.selenium.firefox.XpiDriverService.waitUntilAvailable(XpiDriverService.java:110)
	... 32 more
Caused by: java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(FutureTask.java:205)
	at com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:147)
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)
	... 33 more
]]>
    </failure>
  </testcase> <!-- @BeforeTest setup -->
  <testcase name="@AfterTest end" time="0.0" classname="com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails">
    <skipped/>
  </testcase> <!-- @AfterTest end -->
  <testcase name="verifyLoginWithInvalidCredentails" time="0.001" classname="com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails">
    <skipped/>
  </testcase> <!-- verifyLoginWithInvalidCredentails -->
</testsuite> <!-- Default test -->
